### PR TITLE
new copula_pairs() function

### PR DESCRIPTION
**Pull request by tnagler**
not sure if it makes sense to unit test this function (beyond the example)

### CODE REVIEW

File: R/copula_pairs.R [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+#' @importFrom stats cor
+#' @importFrom graphics box hist.default pairs.default points.default text.default
+#' @export
+#'
+#' @examples
+#' u <- replicate(3, runif(100))
+#' copula_pairs(u)
+copula_pairs <- function(data, ...) {
+  assert_that(is.matrix(data) || is.data.frame(data))
+  assert_that(all(data < 1) && all(data > 0))
```
Comment (tvatter) [inline R/copula_pairs.R]: Would add an assert_that(is.numeric)

Comment (tvatter) [reply R/copula_pairs.R]: (works for matrix, and needs to be applied to every column for data frames)

File: R/copula_pairs.R [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  cor <- call_with_dots(cor, list(x = x, y = y, method = "kendall"), ...)
+  txt <- format(x = cor, digits = 2, nsmall = 2)[1]
+  args <- list(x = 0.5, y = 0.5,
+               labels = txt,
+               cex = 1 + abs(cor) * 3)
+  call_with_dots(text.default, args, ...)
+}
+
+## diagonal panel: histograms (copula data)
+dp_copula_pairs <- function(x, ...) {
```
Comment (tvatter) [inline R/copula_pairs.R]: I would normalize and add an horizontal line for the uniform distribution

File: R/copula_pairs.R (hunk 1 above)
Comment (tnagler) [reply R/copula_pairs.R]: should fail anyway when data is not numeric

**Pull request merged**